bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -161,7 +161,7 @@ public final class Image extends Resource implements Drawable {
  */
 Image (Device device) {
 	super(device);
-	currentDeviceZoom = DPIUtil.getDeviceZoom ();
+	currentDeviceZoom = DPIUtil.getNativeDeviceZoom();
```

Comment:
I think the field should not be called `currentDeviceZoom` anymore, as it suggests it was the actually used device zoom and not the "native" one.